pkg/ocm/clusters.go

```diff
@@ -1193,6 +1223,13 @@ func (c *Client) createClusterSpec(config Spec) (*cmv1.Cluster, error) {
 		clusterBuilder.Autoscaler(BuildClusterAutoscaler(config.AutoscalerConfig))
 	}
 
+	// LogForwarder
+	logFwBuilder := BuildLogForwader(config.LogForwarder)
+	if logFwBuilder != nil {
+		clusterBuilder.ControlPlane(v1.NewControlPlane().
```

Comment:
Looks like this may need to be `cmv1` instead of `v1` - same goes for the line below (after the changes I made in master)